edi_oca/models/edi_backend.py

```diff
@@ -301,10 +302,20 @@ def exchange_send(self, exchange_record):
             _logger.debug("%s sent", exchange_record.identifier)
         except self._send_retryable_exceptions() as err:
             error = _get_exception_msg()
-            _logger.debug("%s send failed. To be retried.", exchange_record.identifier)
-            raise RetryableJobError(
-                error, **exchange_record._job_retry_params()
-            ) from err
+            if self._send_should_retry(exchange_record):
```

Comment:
Thinking out loud...
In theory a case where we spawn another job for the same record should not exist thanks to the `identity_key`.
Hence, I assume that jobs won't be retried infinitely if we reach the max retry job count.
If this does not happen it means the identity_key is not working and we end up w/ more than one job to send the same record.
Am I wrong?